tests/search/search_fixtures/vespa_test_schema/schemas/family_document.sd

```diff
@@ -100,6 +110,45 @@ schema family_document {
         field document_source_url type string {
             indexing: summary | attribute
         }
+
+        field corpus_import_id type string {
```

Comment:
**thought:** Are we duplicating these schema definitions across multiple repositories?